master/internal/api_command.go

```diff
@@ -120,6 +120,12 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 			return nil, launchWarnings, err
 		}
 	}
+	w, err := a.GetWorkspaceByID(ctx, int32(cmdSpec.Metadata.WorkspaceID), *userModel, false)
+	if err != nil {
+		return nil, launchWarnings, fmt.Errorf("retrieving workspace: %w", err)
```

Comment:
I don't think we need to wrap this error. The caller of ```getCommandLaunchParams``` also has enough context added to the error. 

Its not adding any context the error will already be wrapped with something like ```error fetching workspace (%d) from database``` 

so wrapping it would be repetitive with ```retrieving workspace: error fetching workspace (%d) from database: ...``` and it doesn't add any information to the error